test/masq.test.js

```diff
@@ -18,14 +19,25 @@ jest.mock('random-access-idb', () =>
 
 jest.mock('../src/utils', () => {
   const original = require.requireActual('../src/utils')
+  let dbList = {}
   return {
     ...original,
-    dbExists: jest.fn(() => false)
+    dbExists: (name) => {
+      return Promise.resolve(!!dbList[name])
+    },
+    createPromisifiedHyperDB: (name, hexKey) => {
+      dbList[name] = 'db'
```

Comment:
why not store the dbInstance returned by createPromisifiedHyperDB here ?

Reply:
dbList is only meant to make dbExists functional in tests
We do not need to store the instance here